template/flake.nix

```diff
@@ -0,0 +1,55 @@
+{
+  description = "An example of Napalm with flakes";
+
+  # Nixpkgs / NixOS version to use.
+  inputs.nixpkgs.url = "nixpkgs/master";
```

Comment:
`master` tends to break occasionally, `nixpkgs-unstable` is the most suitable branch for projects. Also we should probably avoid flake-registry for now: `github:NixOS/nixpkgs/nixpkgs-unstable`

Reply:
I agree about the `master`, I have no clue why it is there to be honest. 
I am really intrigued, why do you suggest avoiding using registry ? Almost all flakes (if not all) that I have seen assume that you have nixpkgs in your registry, is there a reason why we should avoid it ?

Reply:
In my opion, the main benefit of flakes is that all the dependencies are explicit and contained in the flake. Registry is hidden indirection, which increases the complexity and understandability and makes collaboration harder (person can modify registy, forget about it and then try to update the dependencies), while bringing little benefit (most projects will use locks anyway).